present-day-forecasting/main.py

```diff
@@ -55,6 +55,39 @@
     )
     mcmc.run(jax.random.key(0))
 
+    convergence = get_convergence(
+        mcmc,
+        ignore_nan_in=config["forecasting"]["mcmc"]["convergence"][
+            "ignore_nan_in"
+        ],
+    )
+
+    if (
+        config["forecasting"]["mcmc"]["convergence"]["report_mode"]
+        == "failing"
+    ):
+        convergence = convergence.filter(
+            (
+                pl.col("n_eff")
+                < config["forecasting"]["mcmc"]["convergence"]["ess_cutoff"]
+            )
+            | (
+                pl.col("r_hat")
+                > config["forecasting"]["mcmc"]["convergence"]["ess_cutoff"]
+            )
+        )
+    convergence.write_csv(forecast_dir / f"convergence_{model_name}.csv")
+
+    if (
+        config["forecasting"]["mcmc"]["convergence"]["plot"]
+        and convergence.shape[0] > 0
+    ):
+        plot_dir = forecast_dir / (model_name + "_convergence")
```

Comment:
```suggestion
        plot_dir = forecast_dir / ("convergence_" + model_name)
```

Perhaps, for consistency with the CSVs?